gcp/gcp.go

```diff
@@ -718,8 +742,13 @@ func (gcp *GoogleCloudPrint) assembleJob(job *Job) (*cdd.CloudJobTicket, string,
 
 	gcp.downloadSemaphore.Acquire()
 	t := time.Now()
+	var downloadUrl string
+	if downloadUrl = job.FileURL; !strings.HasPrefix(job.FileURL,"http") {
+		// test env url need to prefix with http
+		downloadUrl = "http://" + job.FileURL
```

Comment:
Why not https? We should definitely be requiring https for print job data shouldn't we?

Reply:
The local GCP server is not in https mode, download url is prefixed only for local GCP servers.